packages/tokens/package.json

```diff
@@ -18,7 +18,7 @@
     "url": "git+https://github.com/chanzuckerberg/edu-design-system.git"
   },
   "scripts": {
-    "build": "style-dictionary build --config=./style-dictionary.config.js",
+    "build": "node ./style-dictionary.config.js",
```

Comment:
since we're not using the json config, should have been doing this to begin with. (also errors in compile in v3) https://amzn.github.io/style-dictionary/#/using_the_npm_module

Reply:
oop good catch